src/AtelierTomato.Markov.Model/Location.cs

```diff
@@ -2,6 +2,6 @@
 {
 	public record Location
 	(
-		IObjectOID ID, string Name
+		IObjectOID ID, string Name, AuthorOID Owner

```

Comment:
still weird to me but i guess. you should think of a strategy to keep all this data reasonably up to date